### PR TITLE
Filenames

### PR DESCRIPTION
**Pull request by jdfreder**
@rgbkrk here's my stab at hiding my Pythonisms.

### CODE REVIEW

**Comment by rgbkrk** [comment]
Because we export all the transformers in the main package, we could make the tests all pull from `../src/transformime` I think.

File: src/transformime.js [hunk 1]
```diff
@@ -104,3 +103,5 @@ export class Transformime {
         return null;
     }
 }
+
+export default {Transformime, TextTransformer, DefaultTransformer, ImageTransformer, HTMLTransformer};
```
Comment (rgbkrk) [inline src/transformime.js]: Tests aren't passing because `TransformerBase` isn't import and exported here.

Comment (jdfreder) [inline src/transformime.js]: thanks

**Comment by rgbkrk** [comment]
:shipit:

**Pull request merged**